week3/src/components/SingleCard.jsx

```diff
@@ -0,0 +1,61 @@
+import cover from "../assets/cover.png";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import "./SingleCard.css";
+
+const SingleCard = ({ handleChoice, data, flipped, disabled }) => {
+  const handleClick = () => {
+    if (!disabled) {
+      handleChoice(data);
+    }
+  };
+  return (
+    <div className="card">
+      <div className={flipped ? "flipped" : ""}>
```

Comment:
이것도 일케 바로 prop으로 넘겨주면된당!!!
```suggestion
      <div flipped={flipped}>
```